pyatlan/client/atlan.py

```diff
@@ -1327,6 +1331,50 @@ def get_lineage_list(
             assets=assets,
         )
 
+    def get_keycloak_events(
+        self, keycloak_request: KeycloakEventRequest
+    ) -> KeycloakEventResponse:
+        raw_json = self._call_api(
+            KEYCLOAK_EVENTS,
+            query_params=keycloak_request.query_params,
+            exclude_unset=True,
+        )
+        if raw_json:
+            try:
+                events = parse_obj_as(list[KeycloakEvent], raw_json)
+            except ValidationError as err:
+                LOGGER.error("Problem parsing JSON: %s", raw_json)
+                raise err
```

Comment:
Using a named expression is more concise:
`        if raw_json := self._call_api(`
`            KEYCLOAK_EVENTS,`
`            query_params=keycloak_request.query_params,`
`            exclude_unset=True,`
`        ):`
`            try:`
`                events = parse_obj_as(list[KeycloakEvent], raw_json)`
`            except ValidationError as err:`
`                LOGGER.error("Problem parsing JSON: %s", raw_json)`
`                raise err`